src/ng-chat/components/ng-chat-window/ng-chat-window.component.html

```diff
@@ -40,45 +40,51 @@
 			</ng-container>
 			<ng-container [ngSwitch]="message.type">
 				<div *ngSwitchCase="MessageType.Text" [ngClass]="{'sent-chat-message-container': message.fromId == userId, 'received-chat-message-container': message.fromId != userId}">
-					<span [innerHtml]="message.message | emojify:emojisEnabled | linkfy:linkfyEnabled"></span>
+          <span [innerHtml]="message.message | emojify:emojisEnabled | linkfy:linkfyEnabled"></span>
+					<span *ngIf="showMessageDate && message.dateSent" class="message-sent-date">{{message.dateSent | date:messageDatePipeFormat}}</span>
+				</div>
+        <div *ngSwitchCase="MessageType.Image" [ngClass]="{'sent-chat-message-container': message.fromId == userId, 'received-chat-message-container': message.fromId != userId}">
+
+          <img src="{{message.message}}" class="responsive-image" />
+
 					<span *ngIf="showMessageDate && message.dateSent" class="message-sent-date">{{message.dateSent | date:messageDatePipeFormat}}</span>
 				</div>
 				<div *ngSwitchCase="MessageType.File" [ngClass]="{'file-message-container': true, 'received': message.fromId != userId}">
-					<div class="file-message-icon-container">
+					<div class="file-message-icon-container">ø
```

Comment:
Is it just me or is there a `ø` character here on this line?

Reply:
sure, I have no idea how that character got here